### PR TITLE
add simple bash completion

### PR DESCRIPTION
**Pull request by scoopex**

### CODE REVIEW

File: resources/bolt_bash_completion.sh [hunk 1]
```diff
@@ -0,0 +1,62 @@
+#####################################################################
+###
+### Bash autocompletion for bolt
+### include this file in your .bash_profile or .bashrc to use completion
+### "source bolt_bash_completion.sh"
+<<<<<<< HEAD
```
Comment (lucywyman) [inline resources/bolt_bash_completion.sh]: It looks like you've still got some git diffs here.

Comment (scoopex) [reply resources/bolt_bash_completion.sh]: i'm sorry :-) Fixed it :-)

Comment (lucywyman) [reply resources/bolt_bash_completion.sh]: You might need to make sure you push to your branch - I'm not  seeing it fixed

**Comment by lucywyman** [comment]
@scoopex Can you squash the 2 commits into 1 commit? And perhaps add more details to your commit message about what you're adding, how to use it, and why? Doesn't need to be long, just a line or 2 :)

```
git rebase -i HEAD~2
```
Then change the "pick" on the second line to "f". Let us know if you need help or want us to do it for ya!

**Pull request merged**